processor/normalizesumsprocessor/processor_test.go

```diff
@@ -417,7 +417,7 @@ func (rmsb resourceMetricsBuilder) addResourceMetrics(resourceAttributes map[str
 	rm := rmsb.rms.AppendEmpty()
 
 	for k, v := range resourceAttributes {
-		rm.Resource().Attributes().Insert(k, v)
+		rm.Resource().Attributes().PutString(k, v.AsString())
```

Comment:
Why didn't we do the same thing here as [agentmetricsprocessor_test.go](https://github.com/GoogleCloudPlatform/opentelemetry-operations-collector/pull/123/files#diff-f44f0ed35b6f181439e5d3caa4bcc8599e08fdf9bfa782f99e0129dc34441e09) (line 150)